enable rattler-build

danielnachun opened the pull request.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

conda-forge-admin commented:
Hi! This is the friendly conda-forge automerge bot!

Commits were made to this PR after the `automerge` label was added. For security reasons, I have disabled automerge by removing the `automerge` label. Please add the `automerge` label again (or ask a maintainer to do so) if you'd like to enable automerge again!

danielnachun committed:
use rattler

danielnachun committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.06.14.27.19

Other tools:
- conda-build 25.11.0
- rattler-build 0.51.0
- rattler-build-conda-compat 1.4.10

danielnachun committed:
use rattler

danielnachun committed:
cross-compile

danielnachun committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.06.14.27.19

Other tools:
- conda-build 25.11.0
- rattler-build 0.51.0
- rattler-build-conda-compat 1.4.10

conda-forge-admin commented:
Hi! This is the friendly conda-forge automerge bot!

I considered the following status checks when analyzing this PR:
 - **linter**: passed
 - **azure**: passed


Thus the PR was passing and merged! Have a great day!
